src/content/h1/tags/hud_message_text/readme.md

```diff
@@ -1 +1,73 @@
-...
+# Creating a proper text file for HUD messages
+This tag is specifically for messages that appear on your HUD. Start by going to the root directory of your level and creating a text file. The contents of the text file differs from the string list above. Instead of using end-string you will define a variable and set the value. Here is an example.
+
+```
+string1=Well, what do we have here?
+string2=If I'm not mistaken this looks like a list of strings.
+string3=A lot of neat things can be done with strings.
+```
+
+Once you've set the contents of the text file make sure to save the file with the encoding set to UTF-16 LE. Make sure the file is named the following...
+
+`hud_messages.hmt`
+
+Tool looks for a file with this name and extension specifically when compiling hud_message tags. On top of needing a specific name, it is also required for the hmt file to reside in the root directory of the scenario the tag is intended for. If we want to import this tag for our tutorial scenario then we will need to save the file at the following location.
```

Comment:
> when compiling hud_message tags

This is a tad misleading because there's no `hud_message` tag, just `hud_message_text`. Better to be clear and explicit rather than use nicknames.